packages/grafana-data/src/types/datasource.ts

```diff
@@ -499,6 +499,7 @@ export interface QueryHint {
 
 export interface MetricFindValue {
   text: string;
+  value?: string | number;
```

Comment:
Maybe we should just go with any here, WDYT?

Reply:
I really think string or number will be the only values that you would use for a select option value.  I don't think an object would work and possibly cause confusion.